pkg/kine/drivers/generic/admission.go

```diff
@@ -4,45 +4,59 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/sirupsen/logrus"
 	"golang.org/x/sync/semaphore"
 )
 
 // AdmissionControlPolicy interface defines the admission policy contract.
 type AdmissionControlPolicy interface {
-	// Admit checks whether the query should be admitted.
+	// admit checks whether the query should be admitted.
 	// If the query is not admitted, a non-nil error is returned with the reason why the query was denied.
 	// If the query is admitted, then the error will be nil and a callback function is returned to the caller.
 	// The caller must execute it after finishing the query
-	Admit(context.Context) (callOnFinish func(), err error)
+	admit(context.Context) (callOnFinish func(), err error)
```

Comment:
I'd argue that it's ok to keep this as exported, no need to change in this PR

Reply:
Done